src/examples/src/widgets/raised-button/ToggleButton.tsx

```diff
@@ -0,0 +1,15 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import RaisedButton from '@dojo/widgets/raised-button';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+
+export default factory(function ToggleButton({ middleware: { icache } }) {
+	const pressed = icache.getOrSet('pressed', false);
```

Comment:
should we be using the generic form of `getOrSet` as a best practice?

Reply:
in this case, the type is implied by the setter @vansimke